<?xml version="1.0"?>
<robot name="MemristorBig">
    <ros2_control name="HardwareControl" type="system">
        <hardware>
            <plugin>mep3_driver::RobotHardwareInterface</plugin>
            <param name="kp_left">100.0</param>
            <param name="ki_left">2.0</param>
            <param name="kd_left">80.0</param>

            <param name="kp_right">100.0</param>
            <param name="ki_right">2.0</param>
            <param name="kd_right">80.0</param>

            <param name="update_rate">50.0</param>
        </hardware>
        <joint name="left_motor">
            <state_interface name="position" />
            <command_interface name="velocity" />
        </joint>
        <joint name="right_motor">
            <state_interface name="position" />
            <command_interface name="velocity" />
        </joint>
    </ros2_control>
</robot>
